README.md

```diff
@@ -336,10 +336,16 @@ evaluation = MTEB(tasks=tasks)
 evaluation.run(
     colbert,
     eval_splits=eval_splits,
+    score_function="max_sim", 
```

Comment:
This would work, but I think we need a more general solution to specify `score_function` to prevent potential future issues with scores

Reply:
Agree -- but probably outside the scope of this PR

Reply:
I completely agree with you. In my opinion, it's a bit surprising that `ModelMeta.similarity_fn_name` isn't being utilized.

Priorities:
1. Pass `score_function` directly to `run()`
2. Utilize `ModelMeta.similarity_fn_name`
3. Default to `RetrievalEvaluator.score_function: str = "cos_sim"`

However, this is currently outside the scope for me...

Reply:
Parameters in `ModelMeta` are only for metadata and don't play a role in processing. I think the solution would be to pass `score_function` to the wrapper, and then `RetrievalEvaluator` can use the parameter from the wrapper

Reply:
> it's a bit surprising that ModelMeta.similarity_fn_name isn't being utilized.

We would love to switch to that one and would encourage a PR for this. It was only recently added to haven't been integrated throughout.